packages/slice-machine/server/src/api/custom-types/save.ts

```diff
@@ -33,6 +33,6 @@ export default async function handler(req: { body: SaveCustomTypeBody }) {
     CustomTypeMockConfig.getCustomTypeMockConfig(updatedMockConfig, model.id)
   );
   // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-  Files.write(mockPath, mocked);
+  Files.write(mockPath, mocked as object);
```

Comment:
weird that we would need to cast again here.

Reply:
not really, generated mocks are of type unknown.

Reply:
That's what I don't get.
In the `mock` function above, it call the mock library, so the types should be known.